lstchain/reco/r0_to_dl1.py

```diff
@@ -178,10 +175,10 @@ def r0_to_dl1(
     calibration_path=None,
     time_calibration_path=None,
     pointing_file_path=None,
-    ucts_t0_dragon=math.nan,
-    dragon_counter0=math.nan,
-    ucts_t0_tib=math.nan,
-    tib_counter0=math.nan
+    ucts_t0_dragon=None,
+    dragon_counter0=None,
+    ucts_t0_tib=None,
+    tib_counter0=None,
```

Comment:
why `None`? should we use `-1` instead?

Reply:
No, None is perfekt in python for "no value given" and is expected like this for by the time calculator

Reply:
This is different for container values because there, the values need to be written to a file using the correct data type, so the defaults must (or rather the missing value indicator, which could be different from the default) must be of the same type. Which is a problem with all data types but floats, since only floats have a natural missing value indicator.